src/entity/MotherCell.js

```diff
@@ -50,13 +50,15 @@ MotherCell.prototype.onUpdate = function () {
             y: this.position.y + size1 * Math.cos(angle)
         };
         
-        // Spawn food
-        var food = new Food(this.gameServer, null, pos, size2);
-        food.color = this.gameServer.getRandomColor();
-        this.gameServer.addNode(food);
-        
-        // Eject to random distance
-        food.setBoost(32 + 42 * Math.random(), angle);
+        if(size1 > this.motherCellMinSize) {
+            // Spawn food
+            var food = new Food(this.gameServer, null, pos, size2);
+            food.color = this.gameServer.getRandomColor();
+            this.gameServer.addNode(food);
+
+            // Eject to random distance
+            food.setBoost(32 + 42 * Math.random(), angle);
```

Comment:
why put 32 + 42? Why not make it:
```javascript
food.setBoost(32 + 42 * Math.random(), angle);
```

Reply:
That's because of the order of operations: it doesn't add those numbers first, then multiply the product. It multiplies 42 by Math.random(), and THEN adds itself to 32. So adding 32 and 42 would make it work incorrectly.

Reply:
Basically BODMAS

Reply:
BODMAS:
```javascript
var x = Math.random();
32 + 42 * x // 35.72347879947521
```
No BODMAS
```javascript
72 * x // 6.383106513386073
``` well you're correct lol